client/gutenberg/extensions/contact-form/editor.js

```diff
@@ -213,6 +213,7 @@ registerBlockType( 'jetpack/field-name', {
 registerBlockType( 'jetpack/field-email', {
 	...FieldDefaults,
 	title: __( 'Email' ),
+	keywords: [ __( 'e-mail' ) ],
```

Comment:
Agree with @lezama here https://github.com/Automattic/wp-calypso/pull/28697#issuecomment-440221289

> `__( 'email' ), 'email',    ` I am probably going to search first for "email" than "correo electronico"

E.g. in Finnish this will get translated to "Sähköposti" for both title and the keyword and hence I wouldn't find it with _"email"_.

Reply:
To avoid that, totally feel free to intentionally avoid translating a keyword if it's international term we want to preserve.